packages/clay-core/src/icon-selector/IconSelector.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * SPDX-FileCopyrightText: © 2025 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayButton from '@clayui/button';
+import Input from '@clayui/form/src/Input';
+import Icon from '@clayui/icon';
+import React, {useState} from 'react';
+
+import DropdownSelector from './DropdownSelector';
+
+export function IconSelector() {
```

Comment:
Shouldn't we receive the same props from the `DropdownSelector` component here to pass to it?

Reply:
I don't think we need it.